Tools/autotest/arducopter.py

```diff
@@ -1675,16 +1678,100 @@ def FenceFloorEnabledLanding(self):
         self.set_rc(3, 1800)
 
         self.wait_mode('RTL', timeout=120)
+
         self.wait_landed_and_disarmed()
         self.assert_fence_enabled()
 
-        # Assert fence is not healthy
+        # Assert fence is healthy now that we have landed disarmed
```

Comment:
You added a "not" here but didn't change the check?  It looks to be checking that the fence is *not* healthy as per the original comment...

Reply:
Fixed